iotedgedev/azurecli.py

```diff
@@ -6,7 +7,7 @@
 from io import StringIO
 from threading import Thread, Timer
 
-from azure.cli.core import get_default_cli
+from azure.cli.core import AzCli, get_default_cli
```

Comment:
>AzCli [](http://example.com/codeflow?start=27&length=5)

why do we need to import AzCli, i am not finding the context where it is used.

Reply:
Removed, forgotten import 😄